flocker_bb/builders/flocker.py

```diff
@@ -434,10 +441,85 @@ def makeOmnibusFactory(distribution, triggerSchedulers=()):
     return factory
 
 
+HOMEBREW_RECIPE_MASTER = "~/Flocker.rb"
+
+
+def makeHomebrewRecipeCreationFactory():
+    factory = getFlockerFactory(python="python2.7")
+    factory.addStep(SetPropertyFromCommand(
+        command=["python", "setup.py", "--version"],
+        name='check-version',
+        description=['checking', 'version'],
+        descriptionDone=['checking', 'version'],
+        property='version'
+    ))
+
+    # Run admin/homebrew.py with BuildBot sdist URL as argument
+    # XXX - how do we know the BuildBot master URL?
+    #
+    # XXX - version of make-homebrew-recipe that can handle URL argument
+    # XXX - is in ClusterHQ/flocker PR #1192
+    factory.addStep(ShellCommand(
+        name='make-homebrew-recipe',
+        description=["building", "recipe"],
+        descriptionDone=["build", "recipe"],
+        command=[
+            "VERSION=1", "admin/make-homebrew-recipe", "some-URL", ">",
```

Comment:
It can be something else, but words get capitalized, dashes get removed. Basically, while it's deterministic, it's tricky to get right and the filename has to match the class name inside the file. So, integers are pretty safe.